package.json

```diff
@@ -4,7 +4,7 @@
   "triggerEmptyDevReleaseByIncrementingThisNumber": 0,
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=14.17",
+    "node": ">=16",
```

Comment:
should this be `16.13.0`?

Reply:
It's not really important here it's only for us in dev but we could, I was going to check .13 in a follow up PR, I'll update these as well

Reply:
I think the important one is in the postinstall hook

Reply:
See https://github.com/prisma/prisma/pull/20148